brickstrap.sh

```diff
@@ -198,10 +198,33 @@ set -o pipefail
 #####################################################################
 
 function create-conf() {
+    #
+    # Set the defaults for mirrors as promised by help, if these haven't been configured for the board yet.
+    #
+    if [ -z "${DEBIAN_MIRROR}" ]; then
+        DEBIAN_MIRROR="http://httpredir.debian.org/debian"
+    fi
+    if [ -z "${RASPBIAN_MIRROR}" ]; then
+        RASPBIAN_MIRROR="http://archive.raspbian.org/raspbian"
+    fi
+    if [ -z "${EV3DEV_MIRROR}" ]; then
+        EV3DEV_MIRROR="http://ev3dev.org/debian"
+    fi
     info "Creating multistrap configuration file..."
     debug "BOARDDIR: ${BOARDDIR}"
     for f in ${BOARDDIR}/packages/*; do
-        while read line; do PACKAGES="${PACKAGES} $line"; done < "$f"
+        while read line; do
```

Comment:
So would I (I just had quite a puzzle as to why that is). I can tinker a bit further with this, but basically it would involve appending (`echo ""`) an additional line and abusing `cat`. Something like: 
```
orig=$(cat $f); 
sanity=$(cat <<<EOF
$(orig)

EOF
);
echo "$(contents)" | while read...
```

Perhaps?